.github/workflows/health-api-library.check.yml

```diff
@@ -1,8 +1,13 @@
 name: Check Health API Library
+
 on:
   push:
     paths:
       - 'health-api-library/**'
+    tags-ignore:
+      - '**'
```

Comment:
🙈what tags are ignored , all?

Reply:
Exactly. The check workflow should not run for tags because we'll start it in the release workflows and only do the release if the check workflow is successful.